<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeaveDetailDialog</class>
 <widget class="QWidget" name="LeaveDetailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Leave Details</string>
  </property>
  <widget class="QLabel" name="employeeInfoHeader">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>740</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Employee Information:</string>
   </property>
  </widget>
  <widget class="QFrame" name="employeeInfoSeparator">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>740</width>
     <height>2</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::HLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Sunken</enum>
   </property>
  </widget>
  <widget class="QLabel" name="empIDLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>User ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="empIDDisplayLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>User12345</string>
   </property>
  </widget>
  <widget class="QLabel" name="appIDLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Application ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="appIDDisplayLabel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>70</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>App98765</string>
   </property>
  </widget>
  <widget class="QLabel" name="leaveDetailsHeader">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>740</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Leave Details:</string>
   </property>
  </widget>
  <widget class="QFrame" name="leaveDetailsSeparator">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>740</width>
     <height>2</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::HLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Sunken</enum>
   </property>
  </widget>
  <widget class="QLabel" name="typeOfLeaveLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Type of Leave:</string>
   </property>
  </widget>
  <widget class="QLabel" name="typeOfLeaveDisplayLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Annual Leave</string>
   </property>
  </widget>
  <widget class="QLabel" name="fromDateLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>From Date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="fromDateDisplayLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2024-11-20</string>
   </property>
  </widget>
  <widget class="QLabel" name="toDateLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>To Date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="toDateDisplayLabel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>230</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2024-11-25</string>
   </property>
  </widget>
  <widget class="QLabel" name="reasonLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Reason:</string>
   </property>
  </widget>
  <widget class="QLabel" name="reasonDisplayLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>600</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Family vacation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="approveButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>400</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Approve</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rejectButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>400</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Reject</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
